lib/celluloid/groups/proactor.rb

```diff
@@ -0,0 +1,56 @@
+require 'thread'
+
+module Celluloid
+  class Group
+    class Proactor < Group
```

Comment:
Yeah, I'm trying for that pattern. The implementation isn't finished yet but there's a finalizer that's not a preset. Actually my default needs to be run in every case but another finalizer can be prepended. Not insisting on the name but am trying to correctly implement this pattern  ( not exclusively as I/O however ).

This gets instantiated then it makes and kills threads, so calling it itself `Thread` seemed wrong, so I'm focusing on its behavior.